packages/rollup-plugin-import-meta-assets/src/rollup-plugin-import-meta-assets.js

```diff
@@ -100,10 +100,12 @@ function importMetaAssets({ include, exclude, warnOnError, transform } = {}) {
               );
               modifiedCode = true;
             } catch (error) {
-              if (warnOnError) {
-                this.warn(error, node.arguments[0].start);
-              } else {
-                this.error(error, node.arguments[0].start);
+              if (error.code !== 'EISDIR') {
```

Comment:
I'd add a comment that this ignores dirs and needs to be done like this to have confidence it's a dir on the file system level

Reply:
actually, you can also make a test by creating a file without an extension and making sure it's processed correctly

Reply:
Done